guides/prow_setup.md

```diff
@@ -1,38 +1,10 @@
 # Prow setup
 
-All Prow related config files for [prow.knative.dev](https://prow.knative.dev)
-are under [config/prod/prow](../config/prod/prow). There is also a staging Prow
-in [prow-staging.knative.dev](https://prow-staging.knative.dev) for testing the
-new config changes before rolling out to production, and all its config files
-are under [config/staging/prow](../config/staging/prow).
-
-## Modify Prow cluster configs
-
 Prow is a collection of microservices that are deployed on a Kubernetes cluster.
 All the Kubernetes object `.yaml` files (except secrets) for Knative production
```

Comment:
We shouldn't need the word `production` here as now we only have one Knative Prow.

Reply:
Done